weeks/week07/README.md

```diff
@@ -636,7 +636,7 @@ events, and JavaScript to create an interactive web page.
 1. Create a **folder** called `cats` on your computer
 1. Create a **file** inside the `cats` folder named `index.html`
 1. Open a **terminal** to your `cats` folder (i.e., `cd cats`)
-1. In your **terminal**, start a web server by running the following command: `npx http-server`
+1. In your **terminal**, start a web server by running the following command: `npx lite-server`
```

Comment:
I'm not sure about this change, because `lite-server` isn't maintained.  `http-server` continues to be maintained and is the most widely used:

https://www.npmtrends.com/http-server-vs-lite-server-vs-serve

Is your thinking to have something that provides live updates?

Reply:
I had an issue on Windows using the ` http-server` on week7 DOM exercise, whereas the `lite-server` worked flawlessly, therefore the proposed change. Perhaps you can leave the document as it is and add a note for students experiencing issues to try with the lite-server as an alternative.

Reply:
Forgive me if this is obvious, but in case it's not clear, you can do more commits on this branch and `push` again to add/update things.

If you need any help with it, let me know.